src/redux/session/actions.js

```diff
@@ -40,10 +45,16 @@ const login = (account, isInitial = false) => dispatch => {
       return dispatch(push('/profile'))
     }
 
-    if (isInitial || !isCBE) { // TODO MINT-108 Non-CBE user should not always start from /wallet route
-      const next = window.localStorage.getItem('next')
-      window.localStorage.removeItem('next')
+    if (isInitial) {
       dispatch(replace(next || ('/' + (!isCBE ? 'wallet' : ''))))
```

Comment:
Probably with this line non-CBE user can access any page using _localStorage.next_ and _LoginPage_.